### PR TITLE
Including scaling (`const_HR_scaling`) in assumed daily capabilities 

### PR DESCRIPTION
**Pull request by marghe-molaro**
This PR allows users to include different “modes” of absenteeism in their scenarios. This was identified as a desired change in Issue #1204.

Each mode corresponds to a table of factors by which daily capabilities of each officer category at different facility levels should be scaled. The mode is specified by the healthsystem module parameter “absenteeism_mode”, and corresponds to a different sheet in the file resources/healthsystem/absenteeism/ResourceFile_Absenteeism.xlsx.

The “default” absenteeism mode, which is specified as the default assumption, assumes a constant factor of 1 for all officer categories, at all facility levels (i.e. ~no absenteeism ever takes place~ no changes relative to assumptions implicit in underlying data). 

The “custom” mode is an illustrative example of how users could customise the assumed factors for theoretical investigations on the impact this would have on the incurred health burden.

The “data” absenteeism mode reflects available data-driven estimates based on the 2018/2019 Malawi Harmonised Health Facility Assessment survey. These factors were obtained by comparing the survey’s processed data on the average minutes worked per day by different officer types at different facility levels (file resources/healthsystem/absenteeism/HHFA_amended_ResourceFile_patient_facing_time.xlsx
 ) with those assumed under the “actual” daily capabilities scenario (resources/healthsystem/human_resources/actual/ResourceFile_Daily_Capabilities.csv). The comparison was carried out using script src/scripts/data_file_processing/healthsystem/human_resources/formatting_absenteeism_factors_from_health_facility_assessment.py

NOTE: Currently, the capabilities obtained from the 2018/2019 Malawi Harmonised Health Facility Assessment survey appear to exceed those assumed in our “actual” scenario in a number of cases (e.g. DCSA). Further discussion will be required before this mode can be included (@wiktafesse).

### CODE REVIEW

**Review by tbhallett — COMMENTED** [review]
Brilliant, thanks Margherita.

Please see on q for your considration about the values for the "data" scenario.

Small changes only suggested: which I've proposed as commits here; but roll back if you prefer something else.

File: src/tlo/methods/healthsystem.py [hunk 1]
```diff
@@ -833,6 +847,12 @@ def pre_initialise_population(self):
 
         # Set up framework for considering a priority policy
         self.setup_priority_policy()
+        
+        # Ensure the mode of absenteeism to be considered in included in the tables loaded
+        assert self.parameters['absenteeism_mode'] in self.parameters['absenteeism_table']
```
[the remark this answers is not in the document]
Comment (tbhallett) [reply src/tlo/methods/healthsystem.py:860]: Actually, now have noticed this comes too later so will propose something else.

File: src/scripts/data_file_processing/healthsystem/human_resources/formatting_absenteeism_factors_from_health_facility_assessment.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import pandas as pd
```
[the remark this answers is not in the document]
Comment (tbhallett) [reply src/scripts/data_file_processing/healthsystem/human_resources/formatting_absenteeism_factors_from_health_facility_assessment.py:1]: Fine! I get it, and sorry for missing your notes.

So maybe the solution is to use a more agnostic naming for this feature (e.g. "officer_minutes_scaler"), that would naturally and obviously accommodate those different options.

[the remark this answers is not in the document]
Comment (marghe-molaro) [reply src/scripts/data_file_processing/healthsystem/human_resources/formatting_absenteeism_factors_from_health_facility_assessment.py:1]: Yes that sounds good - sorry should have named it so from the beginning.

**Comment by marghe-molaro** [comment]
@tbhallett all absenteeism fncs now renamed const_HR_scaling (to differentiate it from dynamic scaling, which I will add to separate PR)

**Review by tbhallett — APPROVED** [review]

**Pull request merged**